### PR TITLE
Add SPI configurations for the standalone mode

### PR DESCRIPTION
**Pull request by style95**
## Description
We need to explicitly configure SPIs for the standalone mode as we updated our default SPI configurations.

## Related issue and scope

- [x] I opened an issue to propose and discuss this change (#5475)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Scheduler
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [x] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Review by dgrove-oss — APPROVED** [review]
lgtm

**Comment by dgrove-oss** [comment]
Would this be enough to allow us to remove https://github.com/apache/openwhisk/blob/master/.github/workflows/4-standalone.yaml#L68-L69 or does that script do more?

**Comment by style95** [comment]
@dgrove-oss 
Thanks for the comment, let me try that.

File: .github/workflows/4-standalone.yaml [hunk 1]
```diff
@@ -65,8 +65,6 @@ jobs:
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
       - name: Check free space
         run: df -h
-      - name: Disable the scheduler
```
Comment (style95) [inline .github/workflows/4-standalone.yaml:68]: It's now obvious that we don't need this step.

**Pull request merged**